contracts/token/ERC721/extensions/ERC721Wrapper.sol

```diff
@@ -29,11 +25,9 @@ abstract contract ERC721Wrapper is ERC721, ERC721Holder {
      * @dev Allow a user to deposit underlying tokens and mint the corresponding tokenIds.
      */
     function depositFor(address account, uint256[] memory tokenIds) public virtual returns (bool) {
-        bytes memory data = abi.encodePacked(WRAPPER_ACCEPT_MAGIC, account);
-
         uint256 length = tokenIds.length;
         for (uint256 i = 0; i < length; ++i) {
-            underlying().safeTransferFrom(_msgSender(), address(this), tokenIds[i], data);
+            underlying().safeTransferFrom(_msgSender(), address(this), tokenIds[i], abi.encodePacked(account));
```

Comment:
What about keeping the existing syntax, and do 
```bytes memory data = abi.encodePacked(account);``` 

That would minimize the changes, and avoid re-encoding multiple time in the loop.

```suggestion
            underlying().safeTransferFrom(_msgSender(), address(this), tokenIds[i], data);
```

Reply:
> avoid re-encoding multiple time in the loop.

Thanks for the heads up! I didn't consider it was inside a loop, just committed